CommandExtensions/builtins/clone_bpd.py

```diff
@@ -96,13 +97,15 @@ def fixup_bpd(cloned: unrealsdk.UObject, known_clones: ClonesDict) -> None:
                 data.Behavior = known_clones[behavior]
                 continue
 
+            if behavior.Class.Name not in cloned_classes:
+                cloned_classes[behavior.Class.Name] = 0
+
             cloned_behavior = clone_object(
-                behavior,
-                cloned,
-                "" if behavior.Name == behavior.Class.Name else behavior.Name
+                behavior, cloned, f"{behavior.Class.Name}_{cloned_classes[behavior.Class.Name]}"
```

Comment:
Specifying a numeric suffix like this is potentially dangerous, since in FNames the string and number are stored separately. You could get a name with the string `my_obj_123` and no number stored, which'd be different to a name `my_obj` with the number 123. Would prefer adding something else (alphabetic) afterwards to to guarantee it's always a name with no number stored.